.github/workflows/tag-and-release.yaml

```
@@ -13,7 +13,7 @@ jobs:
   tag-new-version:
     name: Tag New Version
     permissions: write-all
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.architecture == 'arm64' && 'uds-ubuntu-arm64-4-core' || 'uds-ubuntu-big-boy-8-core' }}
```

```suggestion
    runs-on: ubuntu-latest
```

This doesn't test / publish anything